FEPs/FEP-0001-process.md

```diff
@@ -38,44 +38,49 @@ such matters and making the decision on the way to proceed.
 
 ## Specification
 
-FEPs should be published as Discussions on the FEP repository (TBD). Each FEP is numbered using 4
-digits and new FEPs should get the next free FEP number. FEP numbers are unique and persistent -
-once FEP number is obtained it is reserved for that and only that FEP and cannot be reused later.
+FEPs should be published as Discussions on the FEP repository (TBD).  Each FEP is numbered using
+four digits and new FEPs should get the next free FEP number.  FEP numbers are unique and
+persistent - once a FEP number is obtained, it is reserved for that and only that FEP and cannot be
+reused later.
 
-### Who can make a FEP?
+### Who can create a FEP?
 
-Everyone! Though given that it is very technical document related to the FreeCAD code and/or
-development process it is mostly aimed for being created by the developer community.
+Anyone!  However, given that FEPs are highly technical documents related to the FreeCAD code and/or
+the development process, it is mostly aimed for being created by the developer community.  Although
+not strictly necessary, it is advised that non-developers cooperate with a FreeCAD developer.
```

Comment:
I think that we in the end should make it into strict requirement. Maybe something like that (although probably with better language):
```suggestion
FEPs are highly technical documents related to the FreeCAD code and/or
the development process. Therefore they are mostly aimed for being created by the developer community. Anyone from the GitHub Developers Group can make FEP. If member outside of said group would like to propose FEP it must be done in cooperation with some member of it.
```